libraries/chain/include/eosio/chain/webassembly/eos-vm-oc/memory.hpp

```diff
@@ -50,7 +50,9 @@ class memory {
       // The maximum amount of data that PIC code can include in the prologue
       static constexpr uintptr_t max_prologue_size = mutable_global_size + table_size;
 
-      static_assert(-cb_offset == EOS_VM_OC_CONTROL_BLOCK_OFFSET, "EOS VM OC control block offset has slid out of place somehow");
+      // Changed from -cb_offset == EOS_VM_OC_CONTROL_BLOCK_OFFSET to get around
+      // of compile warning about comparing integers of different signedness
+      static_assert(EOS_VM_OC_CONTROL_BLOCK_OFFSET + cb_offset == 0, "EEOS VM OC control block offset has slid out of place somehow");
```

Comment:
typo introduced in to string here (EEOS)

Reply:
Thanks @spoonincode  for your sharp eyes! Sorry for that.